easybuild/easyconfigs/m/Mono/Mono-4.6.2.7.eb

```diff
@@ -0,0 +1,17 @@
+easyblock = 'ConfigureMake'
+
+name = 'Mono'
+version = '4.6.2.7'
+
+homepage = 'http://mono-framework.com' 
+description = """An open source, cross-platform, implementation of C# and the CLR that is
+ binary compatible with Microsoft.NET."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://download.mono-project.com/sources/mono/']
+sources = [SOURCELOWER_TAR_BZ2]
+
+osdependencies = ['gettext']
```

Comment:
About the dependencies: where do you draw the line? The website http://www.mono-project.com/docs/compiling-mono/linux/ talks mentions " sudo apt-get install git autoconf libtool automake build-essential mono-devel gettext", so this would indicate we also have to add Autotools (which has autoconf, libtool and automake) in as well, right?